waspc/waspc.cabal

```diff
@@ -0,0 +1,382 @@
+cabal-version: 2.4
```

Comment:
@Martinsos if the `waspc.cabal` generated from `package.yaml` was checked into source control on branch `main` before this pull request then we'd see the diff on this file made by moving from stack to cabal.

Reply:
Ha yes, but we didn't have it checked in! It was gitignored.

Why exactly did you mention this, any additional thoughts?